### PR TITLE
Add new api: getDualSolution

### PR DESCRIPTION
**Pull request by chibi314**
Sorry for being late.
This PR is related to #21

### CODE REVIEW

File: include/OsqpEigen/Solver.hpp [hunk 1]
```diff
@@ -37,6 +37,7 @@ namespace OsqpEigen
         Eigen::Matrix<c_float, Eigen::Dynamic ,1> m_primalVariables;
         Eigen::Matrix<c_float, Eigen::Dynamic ,1> m_dualVariables;
         Eigen::VectorXd m_solution;
+        Eigen::VectorXd m_dual_solution;
```
Comment (GiulioRomualdi) [inline include/OsqpEigen/Solver.hpp]: I suggest using the `cammelCase` styling for the class attributes
e.g. 
```cpp
Eigen::VectorXd m_dualSolution;
```

**Comment by chibi314** [comment]
@GiulioRomualdi 
I reflected your suggestion.

**Comment by chibi314** [comment]
Sorry, I rebased master into this branch by mistake.
Currently it's fixed. (rebased devel branch)

**Comment by chibi314** [comment]
The Travis build faild.
I think that's because `devel` is not tracking `master` correctly.

**Pull request merged**